src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -178,6 +178,15 @@
      */
     protected static final String EXON_OUT_INFO_SHORT_NAME = "exonInfo";
 
+    /**
+     * Filter duplicate reads full name
+     */
+    protected static final String FILTER_DUPLICATE_READS_FULL_NAME = "filterduplicatereads";
```

Comment:
typically long names like this would be written with underscores like `filter_duplicate_reads`